requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 git+https://github.com/edx/XBlock.git@9c634481dfc85a17dcb3351ca232d7098a38e10e#egg=XBlock
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@14aeaa9e30f9a408b34ffaf6d78409dedaad015a#egg=edx-submissions==0.1.0
+git+https://github.com/edx/edx-submissions.git@87aaf568400255ad62ef3bebef6a3e6063f700e0#egg=edx-submissions==0.1.0
```

Comment:
To get this to take:

* `pip install -U edx-submissions --force-reinstall`
* `make install`